apps/v1/legacy.go

```diff
@@ -21,7 +20,6 @@ func addLegacyKnownTypes(scheme *runtime.Scheme) error {
 		&DeploymentRequest{},
 		&DeploymentLog{},
 		&DeploymentLogOptions{},
-		&extensionsv1beta1.Scale{},
```

Comment:
The v1 deploymentconfig endpoint uses the extensionsv1beta1 scale, right? Does it also accept the autoscalingv1 scale?  And when this is removed, does the scale endpoint in openshift/apiserver still function properly for all inputs?